mleap-spark/src/main/scala/org/apache/spark/ml/classification/bundle/ops/LinearSVCOp.scala

```diff
@@ -0,0 +1,60 @@
+package org.apache.spark.ml.classification.bundle.ops
+
+import ml.combust.bundle.BundleContext
+import ml.combust.bundle.dsl.{Bundle, Model, NodeShape, Value}
+import ml.combust.bundle.op.OpModel
+import org.apache.spark.ml.bundle.{ParamSpec, SimpleParamSpec, SimpleSparkOp, SparkBundleContext}
+import org.apache.spark.ml.classification.LinearSVCModel
+import org.apache.spark.ml.linalg.Vectors
+
+/**
+  * Class for registering the LinearSVC model from spark.ml project.<br />
+  * Note: Since org.apache.spark.ml.classification.LinearSVCModel is defined as "private[classification]", this class
+  * is defined in a slightly different hierarchy.
+  */
+class LinearSVCOp extends SimpleSparkOp[LinearSVCModel]
+{
+    /** Type class for the underlying model.
+      */
+    override val Model: OpModel[SparkBundleContext, LinearSVCModel] = new OpModel[SparkBundleContext, LinearSVCModel]
+    {
+        override val klazz: Class[LinearSVCModel] = classOf[LinearSVCModel]
+
+        override def opName: String = Bundle.BuiltinOps.classification.linear_svc
+
+        override def store(model: Model, obj: LinearSVCModel)
+                          (implicit context: BundleContext[SparkBundleContext]): Model =
+        {
+            val m = model.withValue("num_classes", Value.long(obj.numClasses))
+            // Set the rest of the parameters
+            m.withValue("coefficients", Value.vector(obj.coefficients.toArray))
```

Comment:
minor, but I think it can be done all in one without the need for local variable m.

Reply:
Combined it into a single line rather than declaring it to an object.